### PR TITLE
phpdoc & composer fixes

### PR DESCRIPTION
**Pull request by Gemorroj**

### CODE REVIEW

**Comment by mikaelcom** [comment]
Impressive! 
How did you fixed PHP docblocks: manually :open_mouth:? using php-cs-fixer?
I'm going to review it quickly!

**Comment by Gemorroj** [comment]
I use PhpStorm. It highlights problems.

File: src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {
     const ATTRIBUTE_MESSAGE = 'message';
     /**
-     * @return TagOperation|null
+     * @return AbstractTag|null
```
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php]: Actually, the returned object should be a TagOperation object as specified by the call
```php
$this->getStrictParent(WsdlDocument::TAG_OPERATION);
```
So I would prefer to let it as it was. it's more accurate.

File: src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ public function getMessage()
         return $message;
     }
     /**
-     * @return TagPart[]|null
+     * @return AbstractTag[]|null
```
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php]: Same comment as for TagOperation

File: src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php [hunk 3]
```diff
@@ -36,7 +36,7 @@ public function getAttributeMessageNamespace()
         return $this->hasAttribute(self::ATTRIBUTE_MESSAGE) ? $this->getAttribute(self::ATTRIBUTE_MESSAGE)->getValueNamespace() : '';
     }
     /**
-     * @return TagMessage
+     * @return AbstractTag|null
```
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php]: As for TagOperation, it should returna TagMessage thanks to the `'message'` parameter and the fact that [`WsdlToPhp\PackageGenerator\DomHandler\Wsdl\AbstractDocument::getElementHandler`](https://github.com/WsdlToPhp/PackageGenerator/blob/develop/src/DomHandler/Wsdl/AbstractDocument.php#L76) method uses the class corresponding to the tag name

File: src/DomHandler/Wsdl/Wsdl.php [hunk 4]
```diff
@@ -78,7 +78,12 @@ public function getElementsByName($name, $includeExternals = false)
             $name,
         ), $includeExternals);
     }
+
```
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Wsdl.php]: there is not blank line anywhere, could you remove it?

**Comment by mikaelcom** [comment]
good to know about PhpStorm ! :smile:

File: src/DomHandler/Wsdl/Wsdl.php (hunk 4 above)
Comment (Gemorroj) [inline src/DomHandler/Wsdl/Wsdl.php]: http://www.php-fig.org/psr/psr-2/#2-2-files
>> All PHP files MUST end with a single blank line.

File: src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php [hunk 5]
```diff
@@ -36,7 +36,7 @@ public function getAttributeMessageNamespace()
         return $this->hasAttribute(self::ATTRIBUTE_MESSAGE) ? $this->getAttribute(self::ATTRIBUTE_MESSAGE)->getValueNamespace() : '';
     }
     /**
-     * @return AbstractTag|null
+     * @return TagOperation|null
```
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php]: Actually it's a TagMessage :wink:, sorry

File: src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php [hunk 6]
```diff
@@ -50,7 +50,7 @@ public function getMessage()
         return $message;
     }
     /**
-     * @return AbstractTag[]|null
+     * @return TagOperation[]|null
```
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Tag/AbstractTagOperationElement.php]: Actually it's a TagPart[] here :wink:, sorry

File: src/DomHandler/Wsdl/Wsdl.php (hunk 4 above)
Comment (mikaelcom) [inline src/DomHandler/Wsdl/Wsdl.php]: Indeed but [line 81](https://github.com/WsdlToPhp/PackageGenerator/pull/70/files/8ed83e77bf3b553721664e028173a34051e73af9#diff-f2a43b16493644355096c1c7656a8984L80) is not the end of the file :wink:

**Comment by mikaelcom** [comment]
Are you finish or are you planning to finish it later?
Thx

**Comment by Gemorroj** [comment]
@mikaelcom I finished, at the moment.

**Pull request merged**